assets/styles/slug-pages.scss

```diff
@@ -107,6 +107,16 @@
       top: -70px; //offset links in the document by the height of the sticky header
     }
   }
+
+  :deep(figure) {
```

Comment:
Do we need this? If we do, this needs more specificity. It's breaking the spacing for other components like the card with image flexible page block
<img width="840" height="666" alt="Screenshot 2025-10-31 at 9 57 52 AM" src="https://github.com/user-attachments/assets/b9f92c2b-b289-4f31-acb7-03922588cd68" />